packages/happy-css-modules/src/test-util/util.ts

```diff
@@ -10,7 +10,7 @@ import { sleepSync } from '../util.js';
 export const FIXTURE_DIR_PATH = resolve(
   realpathSync(tmpdir()),
   'happy-css-modules/fixtures',
-  process.env['JEST_WORKER_ID']!,
+  process.env['VITEST_WORKER_ID']!,
```

Comment:
MAY: `VITEST_WORKER_ID` is also fine, but `VITEST_POOL_ID` is better. `VITEST_POOL_ID` is an environment variable that has the same meaning as `JEST_WORKER_ID`.

https://vitest.dev/guide/migration#envs

Reply:
Replaced with VITEST_POOL_ID in [846bbb8](https://github.com/mizdra/happy-css-modules/pull/288/commits/846bbb8cf2041828305654cc5ea9a1d088cb6aa6).